api/remove-duplicate-version-badge.js

```diff
@@ -6,7 +6,7 @@ function removeDuplicateVersionBadge(generatedInfoFilePath) {
   console.log("removing duplicate version badge...");
   replace.sync({
     files: generatedInfoFilePath,
-    from: /^.*Version: .*$/m,
```

Comment:
This regex changes, because the API docs generator plugin generates different markup now. 

Previously, the entire element for the version badge was on one line; now it is spread across 4 lines, with a span element surrounding the version number.